library/src/main/java/com/telefonica/tweaks/domain/tweakModels.kt

```diff
@@ -41,10 +41,10 @@ data class TweakCategory(val title: String, val groups: List<TweakGroup>) {
 
         fun group(
             title: String,
-            withClearButton: Boolean = true,
+            hideResetButton: Boolean = false,
```

Comment:
I think we can avoid making this breaking change by leaving the flag as it is. We can change your new condition to something like this:
```
tweakGroup.entries.any { it is Editable<*> } && tweakGroup. withClearButton
```
We would keep old behavior and current client implementations. Also we would avoid scenarios where old usages are declared without named params where the functionality would have been mistakenly inverted
```
// old version
group("title", false) // -> we don't want to show the button and it isnt' shown

// new version
group("title", false) // -> we don't want to show the button BUT it is shown
```

Please correct me if I'm wrong 😅

Reply:
You are totally right in how it was, and now it is working.

IMO, I thought that making a breaking change in favor to have something a little bit more intuitive when implementing it and avoid unnecessary reset buttons could be worth.

Right now in our projects I doubt that parameter was being used. But thinking in everyone else, what you are saying makes sense.

I dunno if would be "enough" clarify this change in release notes for everyone.

I don't have a strong opinion but I'm not totally confortable keeping old behaviour, anyway as I said I'm all ears 🦻🏼

Reply:
> and avoid unnecessary reset buttons could be worth

But the client still needs to set a flag in order to hide the button right? the new default behavior is the same as the previous one, except that the button won't appear if there isn't any editable tweak even if the flag says so

Reply:
I don't have a strong opinion either, but I'm with Yamal in this one, I think it's more convenient to don't have such a breaking change in this case

Reply:
Fair enough, undoing breaking change, in 3... 2... 1...
Thanks you both for your advice!